arbnode/batch_poster.go

```diff
@@ -2558,6 +2493,238 @@ func (b *BatchPoster) MaybePostSequencerBatch(ctx context.Context) (bool, error)
 	return true, nil
 }
 
+func (b *BatchPoster) getL1Bounds(ctx context.Context) (*l1Bounds, error) {
```

Comment:
instead of doing these conditional if statements why cant we just create two get functions one for normal l1Bounds and one for timeboost?